spec/controllers/reviewers/applications_controller_spec.rb

```diff
@@ -95,8 +95,8 @@
 
         before { get :show, params: { id: application } }
 
-        it 'assigns @mentor_comments' do
-          expect(assigns :mentor_comments).to contain_exactly(mentor_comment)
+        it 'shows mentor comment' do
+          expect(response.body).to match(mentor_comment.text)
```

Comment:
💟 

One minor (optional) thing: can you change the `text` value for the record that's being created in line 94 to something more specific so that we don't accidentally match on a string that's present through other means?

Reply:
Sounds good! Will do.